Copy to SD Card root directory to update/config.ini

```diff
@@ -231,12 +231,12 @@ fan_ctrl_count:0
 max_temp:T0:275 T1:275 T2:275 T3:275 T4:275 T5:275 BED:150 CHAMBER:60
 
 #### Cold Extrusion Minimum Temperature
-# Minimum temperature need to extrude/retract. Any extrude/retract below this temperature will be prevented.
+# Minimum temperature need to extrude/retract. Any extrude/retract below this temperature will be prevented
```

Comment:
Same here. Periods are used to end sentences, so why not keep them?

Reply:
It is removed always on the last sentence. So if you have only one sentence, "." is not used (as it was in almost all the config.ini file). If there are more sentences (as in one you reported), "." is removed on the last one (it means there is no continuation)

Reply:
Periods should be added instead of removing them to make the sentence look complete. why would there be a pattern for not having the periods on the last sentence? if it is missing somewhere then it should be added not removed from other parts. Deriving patterns from everything is not a great way to follow. Somethings look better when they are a little bit out of order.

Reply:
> Periods should be added instead of removing them to make the sentence look complete.

I agree. Periods end sentences and removing them here is odd.

Reply:
It is not a problem at all for me to adopt one solution or another but it should be applied uniformly on the whole file. In most part of the file it was omitted. If we should make the file more readable it should be most important to use a separation between a description and param format qualifier instead of scratching everything as it is now